src/main/java/com/minecolonies/coremod/entity/ai/citizen/miner/EntityAIStructureMiner.java

```diff
@@ -603,22 +604,22 @@ private void initStructure(final Node mineNode, final int rotateTimes, BlockPos
     {
         if(mineNode == null)
         {
-            loadStructure("miner/minerMainShaft", getRotationFromVector(), structurePos, false);
+            loadStructure(Structures.SCHEMATICS_PREFIX + "/miner/minerMainShaft", getRotationFromVector(), structurePos, false);
         }
         else
         {
             //todo we can add other nodeTypes without a problem.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)